fe/src/main/java/org/apache/doris/analysis/ArithmeticExpr.java

```diff
@@ -133,6 +135,11 @@ public ArithmeticExpr(Operator op, Expr e1, Expr e2) {
         if (e2 != null) {
             children.add(e2);
         }
+        if (children.size() == 1) {
```

Comment:
Why generating sqlStr? I think it will make all objects generate an string, in some case that toSql is not called, sqlStr wouldn't be generated.

Reply:
the expr after analyze will be changed, this will cause output table header and user input sql are different e.g. 
![image](https://user-images.githubusercontent.com/9098473/70196184-e2e28d80-1742-11ea-9b32-fbc4f8bb9db5.png)
generating sqlStr at constructor will keep the origin sql 
![image](https://user-images.githubusercontent.com/9098473/70196243-18877680-1743-11ea-9444-33adbab04a8e.png)